app/src/main/java/com/example/demomvp/ui/task/TaskAdapter.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.demomvp.ui.task
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.demomvp.R
+import com.example.demomvp.data.model.Task
+
+class TaskAdapter(private val context: Context) : RecyclerView.Adapter<TaskAdapter.ViewHolder>() {
+
+    private var mListTask: ArrayList<Task> = ArrayList()
```

Comment:
Tại sao lại là var
trong kotlin đặt tên biến dạng camelCase
```suggestion
    private val tasks = mutableListOf()
```